### ISSUE TITLE
The example code errors

### ISSUE THREAD

**Issue by yaquawa** [lapwinglabs/x-ray-phantom#26]
I got this when I run the example code. (The version of my node.js is v8.5.0)

```
phantom stdout: NETWORK_ERR: XMLHttpRequest Exception 101: A network error occurred in synchronous requests.

  phantomjs://code/shim.js:1815 in send

phantom stdout:   phantomjs://code/shim.js:1815 in _start
  phantomjs://code/shim.js:1842
```

**Comment by TaylorAckley** [comment]
Cosign, though im pretty sure this is on Phantom.   Here is my code:

```
const x = Xray()
    .driver(phantom({
        XSSAuditingEnabled: true,
        loadImages: false,
        webSecurityEnabled: false
    }))
    .concurrency(3)
    .throttle(3, 500);
```

**Comment by TaylorAckley** [comment]
@yaquawa I forgot to check the closed issues.... see here for resolution:

(Note the `webSecurity:false` prop passed to the phantom constructor)

https://github.com/lapwinglabs/x-ray-phantom/issues/16

**Comment by knandraina** [comment]
@TaylorAckley Where did you put this code? Into a new file?

**Comment by TaylorAckley** [comment]
@knandraina In reference to what?   Which code?

**Comment by knandraina** [comment]
@TaylorAckley Yes I should have been more accurate. I have the same error as above 

`phantom stdout: NETWORK_ERR: XMLHttpRequest Exception 101: A network error occurred in synchronous requests.`
`  phantomjs://code/shim.js:1815 in send`
`phantom stdout:   phantomjs://code/shim.js:1815 in _start`
`  phantomjs://code/shim.js:1842`

I would like to fix that mistake and I asked where I needed to put this code 

`const x = Xray()`
      `.driver(phantom({`
         ` XSSAuditingEnabled: true,`
          `loadImages: false,`
         ` webSecurityEnabled: false`
      `}))`
     ` .concurrency(3)`
      `.throttle(3, 500);`

**Comment by TaylorAckley** [comment]
It would replace the XRay instantiation you already have.

I guess if you need an example you can use my partially completed program: https://github.com/TaylorAckley/SitemapScraper/blob/master/Scraper.js

**Comment by ralphdas** [comment]
Watch out for the exact wording here: 

```const Scraper = Xray()
    .driver(phantom({
        XSSAuditingEnabled: true,
        loadImages: false,
        webSecurity: false
    }))  
```

it is webSecurity not webSecurityEnabled

**Comment by kjr247** [comment]
I also get this issue.
